### PR TITLE
Setup project and enabled CircleCI

### PR DESCRIPTION
**Pull request by preetitheraja**
Added a Springboot application.

### CODE REVIEW

**Comment by shekhargulati** [comment]
@preetitheraja why you closed this PR?

File: README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ A web application to manage OKR.
 ## Resources
 
 1. :memo: Introduction to OKR - [Link](https://www.oreilly.com/business/free/files/introduction-to-okrs.pdf)
+[![CircleCI](https://circleci.com/gh/fs-101/okr-manager.svg?style=svg)](https://circleci.com/gh/fs-101/okr-manager)
```
Comment (shekhargulati) [inline README.md]: Move badge to the top. See this demo repo https://github.com/CircleCI-Public/circleci-demo-ios

**Review by shekhargulati — CHANGES_REQUESTED** [review]
You will also need to commit .mvn folder. It contains the wrapper jar. It is required to build the application

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2 # use CircleCI 2.0 
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+  
+    docker: # run the steps with Docker
+      - image: circleci/openjdk:8-jdk-stretch # ...with this image as the primary container; this is where all `steps` will run
+      
+    steps: # a collection of executable commands
+      - checkout
+      - run: mvn clean compile
```
Comment (shekhargulati) [inline .circleci/config.yml]: change this to `./mvnw clean install`

Also learn why we need to do this.

**Review by shekhargulati — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (preetitheraja) [reply .circleci/config.yml]: done

File: README.md (hunk 1 above)
Comment (preetitheraja) [reply README.md]: done

**Comment by preetitheraja** [comment]
> You will also need to commit .mvn folder. It contains the wrapper jar. It is required to build the application

**Review by shekhargulati — APPROVED** [review]

**Pull request merged**